src/xcode/ENA/ENA/Source/AppDelegate.swift

```diff
@@ -83,10 +83,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 		NotificationCenter.default.addObserver(self, selector: #selector(isOnboardedDidChange(_:)), name: .isOnboardedDidChange, object: nil)
 		NotificationCenter.default.addObserver(self, selector: #selector(backgroundRefreshStatusDidChange), name: UIApplication.backgroundRefreshStatusDidChangeNotification, object: nil)
 
+		checkForRisks()
 		return true
 	}
 
 	func applicationWillEnterForeground(_ application: UIApplication) {
+		checkForRisks()
```

Comment:
Looks like this will try to trigger the Riskcalculation twice on a fresh Appstart. I think it would make sense to move/copy `let detectionMode = DetectionMode.fromBackgroundStatus()
 		riskProvider.riskProvidingConfiguration.detectionMode = detectionMode		 		 to the 
`	lazy var riskProvider: RiskProvider = {
		let keyPackageDownload = KeyPackageDownload(
			downloadedPackagesStore: downloadedPackagesStore,
			client: client,
			wifiClient: wifiClient,
			store: store
		)`